index.js

```diff
@@ -182,6 +182,8 @@ EasyXml.prototype.parseChildElement = function(parentXmlNode, parentObjectNode)
                     } else {
                         parentXmlNode.set(key.substring(1), child);
                     }
+                } else if (typeof child === 'object' && typeof child.toString === 'function') {
```

Comment:
I ran into problem serializing Mongodb objectIds, they look like strings but in order to get their string representation you need to call their toString method. see https://docs.mongodb.org/v3.0/reference/method/ObjectId.toString/